### DISCUSSION TITLE
How to add two Py5Vectors

### DISCUSSION THREAD

**Discussion by vsquared** [py5coding/py5generator#544] [Q&A] [answered]
I'm trying to port a Processing app to Py5 and want to add two Py5Vectors together.  The following causes an error shown below:
```
loc = Py5Vector(width/2,height/2)
polar = Py5Vector(r*cos(theta),r*sin(-theta))
end = Py5Vector.add(loc,polar(w,orientation))
```
<img width="246" alt="e" src="https://github.com/user-attachments/assets/fb3ee520-880d-4918-9e67-883aeda3e53c">

**Comment by Maltiec** [comment, marked as answer]
just use `+`:   `end = loc + polar`

**Reply by vsquared** [reply]
Thank you!
